about/about-network-policy.md

```diff
@@ -0,0 +1,223 @@
+
+To learn more about Calico network policies, read the [Get started with Calico network
+policy]({{site.baseurl}}/security/calico-network-policy) guide.
+
+### Benefits of using {{site.prodname}} for network policy
+
+#### Full Kubernetes network policy support
+Unlike some other network policy implementations, Calico implements the full set of Kubernetes network policy features.
+
+#### Richer network policy
+Calico network policies allow even richer traffic control than Kubernetes network policies if you need it. In addition,
+Calico GlobalNetworkPolicy allows you to create policy that applies across multiple namespaces.
+
+#### Mix Kubernetes and Calico network policy
+Kubernetes and Calico network policies can be mixed together seamlessly. One common use case for this is to split
+responsibilities between security / cluster ops teams and developer / service teams. For example, giving the security /
+cluster ops team RBAC permissions to define Calico policies, and giving developer / service teams RBAC permissions to
+define Kubernetes network policies in their specific namespaces. As Calico policy rules can be ordered to be enforced
+either before or after Kubernetes network policies, and can include actions such as deny and log, this allows the
+security / cluster ops team to define basic higher-level more-general purpose rules, while empowering the developer /
+service teams to define their own fine-grained constraints on the apps and services they are responsible for.
+
+For more flexible control and delegation of responsibilities between two or more teams, Calico Enterprise extends this
+model to provide [hierarchical policy](#hierarchical-policy).
+
+![Example mix of network policy types]({{site.baseurl}}/images/example-k8s-calico-policy-mix.svg)
+
+#### Ability to protect hosts and VMs
+As {{site.prodname}} policies can be enforce on host interfaces, you can use them to protect your Kubernetes nodes (not
+just your pods), including for example, limiting access to node ports from outside of the cluster. To learn more, check
+out the {{site.prodname}} [policy for hosts]({{site.baseurl}}/security/hosts) guides.
+
+#### Integrates with Istio
+When used with Istio service mesh, {{site.prodname}} policy engine enforces the same policy model at the host networking
+layer and at the service mesh layer, protecting your infrastructure from compromised workloads and protecting your
+workloads from compromised infrastructure. This also avoids the need for dual provisioning of security at the service
+mesh and infrastructure layers, or having to learn different policy models for each layer.
+
+#### Extendable with Calico Enterprise
+[Calico Enterprise]({{site.baseurl}}/calico-enterprise/) adds even richer network policy capabilities, with the ability
+to specify hierarchical policies with each team have particular boundaries of trust, and FQDN / domain names in policy
+rules for restricting access to specific external services.
```

Comment:
This section seems like it should go in Concepts section of Get started with Calico network policy...where the user is actually doing the task. We could also use an include and add to both.